docs/design.md

```diff
@@ -18,8 +18,8 @@ Firecracker virtual machine manager (VMM).
 
 1. Firecracker can safely run workloads from different customers on the same
    machine.
-1. Customers can create microVMs with any combination of vCPU and memory to
-   match their application requirements.
+1. Customers can create microVMs with any combination of vCPU (up to 32)
```

Comment:
where does this restriction come from? Maybe we should add a comment?

Reply:
The reason for this is mentioned [here](https://github.com/firecracker-microvm/firecracker/blob/firecracker-v1.1/src/vmm/src/vmm_config/machine_config.rs#L9).

I am not sure how to rephrase this to also include that especially because the first paragraph already mentions the lightweight microVM story. I am opened to suggestions.

Reply:
Ok, I see. To me, it's not a clear the distinction `lightweight VM <=> up to 32 vCPUs`, but I don't see a way out of this now.